.github/workflows/e2e.yaml

```diff
@@ -41,7 +44,7 @@ jobs:
     - name: Start clusters
       working-directory: test
       run: |
-        drenv start --name-prefix ${{ env.NAME_PREFIX }} envs/regional-dr.yaml
+        drenv start --max-workers ${{ env.MAX_WORKERS }} --name-prefix ${{ env.NAME_PREFIX }} envs/regional-dr.yaml
```

Comment:
extend the dev-env.sh script to get the MAX_WORKERS env var and call drenv accordingly.

Reply:
That wrapper scripts is not helpful. I don't mind if you want to use it locally but it should not be sued in the CI.